edb/schema/functions.py

```diff
@@ -667,17 +666,27 @@ def is_blocking_ref(
         return not isinstance(reference, Parameter)
 
 
-class CallableCommand(sd.QualifiedObjectCommand):
+class CallableCommand(sd.QualifiedObjectCommand[CallableObject]):
 
-    def _get_params(self, schema: s_schema.Schema, context):
+    def _get_params(
+        self,
+        schema: s_schema.Schema,
+        context: sd.CommandContext
+    ) -> so.ObjectList[Parameter]:
         params = []
         for cr_param in self.get_subcommands(type=ParameterCommand):
-            param = schema.get(cr_param.classname)
+            param: Parameter = schema.get(cr_param.classname)
```

Comment:
```suggestion
            param = schema.get(cr_param.classname, type=Parameter)
```

This spelling is better because it also performs runtime checking of the actual returned object.